service/chartconfig/v1/helm/tunnel.go

```diff
@@ -0,0 +1,107 @@
+package helm
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"strconv"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/portforward"
+	"k8s.io/client-go/transport/spdy"
+)
+
+// tunnel describes a ssh-like tunnel to a kubernetes pod
```

Comment:
Great that there is GoDoc here but nitpick that there should be `.`.

Reply:
Sure thx, I've fixed all GoDoc comments.